test/Microsoft.Azure.ServiceBus.UnitTests/Diagnostics/QueueClientDiagnosticsTests.cs

```diff
@@ -28,7 +28,7 @@ async Task EventsAreNotFiredWhenDiagnosticsIsDisabled()
             this.listener.Disable();
 
             await TestUtility.SendMessagesAsync(this.queueClient.InnerSender, 1);
-            this.queueClient.RegisterMessageHandler((msg, ct) => {
+            this.queueClient.RegisterMessageHandler((Message msg, CancellationToken ct) => {
```

Comment:
I'm not sure we need to spell out the types. Though if you do want to provide better names for the arguments to indicate what they stand for then 👍

Also, this kind of change would be probably better to have as a separate PR to separate the concerns and leave the focus on batched receive.